ui/src/data_explorer/containers/DataExplorer.js

```diff
@@ -90,7 +88,7 @@ const DataExplorer = React.createClass({
           autoRefresh={autoRefresh}
           timeRange={timeRange}
         />
-        <ResizeContainer>
+        <ResizeContainer containerClass="page-contents" minTopHeight={350}>
```

Comment:
350 feels like a "magic number" -- could this be done via a special class passed in so that we have better inline semantic clarity? otherwise it feels like an arbitrary and fickle value being passed in, to me (aka I agree with what I perceive to be the intention behind the `no-magic-numbers` eslint rule.)

that being said, i'm happy to see this being done via classes rather than a special `<ResizeBottom>` component